data/mods/DinoMod/overmap/map_extras.json

```diff
@@ -265,11 +265,11 @@
     "flags": [ "CLASSIC" ]
   },
   {
-    "id": "mx_nest_einiosaurus",
+    "id": "mx_nest_euoplocephalus",
```

Comment:
Looks like `mx_nest_einiosaurus` is being unintentionally overwritten?
In my game I got an error message about missing `id` while testing.

Reply:
Good eye, thanks!